spaceship/lib/spaceship/portal/certificate.rb

```diff
@@ -204,13 +207,30 @@ def create_certificate_signing_request
           csr = OpenSSL::X509::Request.new
           csr.version = 0
           csr.subject = OpenSSL::X509::Name.new([
-                                                  ['CN', 'PEM', OpenSSL::ASN1::UTF8STRING]
+                                                  ['CN', 'spaceship', OpenSSL::ASN1::UTF8STRING]
```

Comment:
Do we want to or can we change this unrelated method here in this PR? Probably not.

Reply:
Ok, makes sense here - this was not correct before.